### DISCUSSION TITLE
How do I dynamically change the Nutgram bot token in Laravel?

### DISCUSSION THREAD

**Discussion by loopeen** [nutgram/nutgram#186] [Q&A]
My database has a table with bots in which their tokens are stored, and the webhook specifies the parameter ID, which stores the id of the bot in the database, I need to get the bot from the database by this id, take the token and change it in Nutgram. Currently Nutgram takes the token from .env

**Comment by sergix44** [comment]
You should probably create and register your own service provider that creates a nutgram instance with the token you provide, and put nutgram in the dont-discovery part so only your provider is going to be registered.
